geode/content.md

```diff
@@ -0,0 +1 @@
+Please refer https://hub.docker.com/r/apachegeode/geode/
```

Comment:
This is going to need to be much more verbose before we can consider merging, btw. :wink:

(Figure this was probably just intended as a temporary placeholder while https://github.com/docker-library/official-images/pull/3685 was evaluated.)